.github/workflows/webinstall.yaml

```diff
@@ -0,0 +1,33 @@
+name: tools/webinstall
+
+on:
+  push:
+    branches: [stable]
+    paths:
+      - 'tools/webinstall/**'
+
+jobs:
+  myself-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Registry
+        if: ${{ github.event_name == 'push' }}
+        uses: docker/login-action@v3
+        with:
+          registry: index.unikraft.io
+          username: ${{ secrets.REG_USERNAME }}
+          password: ${{ secrets.REG_TOKEN }}
+
+      - name: Build & Push OCI Image
+        uses: docker/build-push-action@v5
+        if: ${{ github.event_name == 'push' }}
+        with:
+          push: true
+          tags: index.unikraft.io/kraftkit-internal/webinstall:latest
```

Comment:
Better avoid pushing to Harbor if we can.

Reply:
why? This is something I anyway do manually

Reply:
Because when we upgrade Kubernetes and Harbor is down, no application can be deployed and we have a downtime.